bin/main.rb

```diff
@@ -94,57 +56,77 @@ def play(player1, player2)
   end
 end
 
-def results
-  # needs finnesing
+def score(player1, player2)
+  if @board.win == 1
+    player1.score += 1
+    player1.name.green
+  elsif @board.win == 2
+    plyer2.score += 1
+    player2.name.blue
+  else
+    false
+  end
+end
```

Comment:
- [ ] this method should be moved to the `game` class. This is related to `game logic` than user-interface logic